File: SoloLearn Codes/Popsicles/Popsicles.cs
using System;


namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            int siblings = Convert.ToInt32(Console.ReadLine());
            int popsicles = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine(popsicles % siblings == 0 ? "give away" : "eat them yourself");
        }
    }
}

File: SoloLearn Codes/Popsicles/Popsicles.java
import java.util.Scanner;

public class Popsicles
{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int siblings = sc.nextInt();
        int popsicles = sc.nextInt();
        System.out.println(popsicles % siblings == 0 ? "give away" : "eat them yourself");
    }
}